cli/config.go

```diff
@@ -619,6 +623,18 @@ func loadConfiguration(configPath string, ignoreLogging bool) (resolvedConfigPat
 	return resolved, config, nil
 }
 
+func checkNetNsConfig(config oci.RuntimeConfig) error {
```

Comment:
I like this function :)
Please add a comment explaining what it does as this is an important sanity check of the configuration file.

Reply:
Done